internal/lvmd/command/lvm_command_test.go

```diff
@@ -192,5 +192,39 @@ func Test_lvm_command(t *testing.T) {
 				t.Fatalf("expected error to be %v, got %v", ErrNoMultipleOfSectorSize, err)
 			}
 		})
+
+		t.Run("create cached volume and read it", func(t *testing.T) {
+			// create the cachedevice
+			cache_vg_name := "CACHEDEVICE"
+			cache_loop, err := testutils.MakeLoopbackDevice(ctx, cache_vg_name)
+			if err != nil {
+				t.Fatal(err)
+			}
+			err = testutils.MakeLoopbackVG(ctx, cache_vg_name, cache_loop)
+			if err != nil {
+				t.Fatal(err)
+			}
+			// ensure cache device cleanup
+			defer func() { _ = testutils.CleanLoopbackVG(cache_vg_name, []string{cache_loop}, []string{cache_vg_name}) }()
+			vg, err := FindVolumeGroup(ctx, cache_vg_name)
+			if err != nil {
+				t.Fatal(err)
+			}
+			// create cached LV
+			err = vg.CreateVolume(ctx, "test1", uint64(topolvm.MinimumSectorSize), []string{"tag"}, 0, "", []string{"--type", "writecache", "--cachesize", "10M", "--cachedevice", cache_loop})
+			if err != nil {
+				t.Fatal(err)
+			}
+			vol, err := vg.FindVolume(ctx, "test1")
+			if err != nil {
+				t.Fatal(err)
+			}
+			if vol.IsThin() {
+				t.Fatal("Expected test1 to not be thin (eval lv_attr instead of pool)")
+			}
+			if !strings.Contains(vol.attr, "C") {
```

Comment:
```suggestion
			if vol.attr[0] != byte(VolumeTypeCached) {
```
I want to avoid using 'contain' because it may misidentify other fields.